concent_api/core/tests/test_unit_message_handlers.py

```diff
@@ -2,8 +2,13 @@
 
 from django.conf import settings
 from django.test import override_settings
+
+from common import constants
```

Comment:
You are actually using only `ErrorCode` from `constants` so why not importing it directly ?

Reply:
good point, done